civihr_default_theme/scss/base/_custom.scss

```diff
@@ -1235,3 +1235,7 @@ form {
     }
   }
 }
+
+.pointer {
```

Comment:
No code should be added to _custom.css as it contains all the legacy css code that one day should be deleted completely. Please find a suitable partial (or create one) for this kinda of utility classes